common/include/pcl/experimental/executor/blocking.cpp

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2014-, Open Perception, Inc.
+ *  Author(s): Shrijit Singh <shrijitsingh99@gmail.com>
+ *
+ */
+#include <pcl/experimental/executor/property.h>
+
+namespace executor {
+constexpr blocking_t::possibly_t blocking_t::possibly;
+constexpr blocking_t::always_t blocking_t::always;
+constexpr blocking_t::never_t blocking_t::never;
+}  // namespace executor
```

Comment:
Once C++17 support kicks in, I would declare these inline and move them of the appropriate header.  May be add a `\todo` note so that it pops in [doxygen](http://pointclouds.org/documentation/todo.html).

Reply:
Done